public/resources/remix/index.js

```diff
@@ -6,122 +6,136 @@
 
   var gaTrackingId = 'UA-68630113-1';
 
-  function injectAnalytics($) {
-    var analyticsHtml =
-    '\n<script>\n' +
-    '(function(i,s,o,g,r,a,m){i[\'GoogleAnalyticsObject\']=r;i[r]=i[r]||function(){\n' +
-    '(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),\n' +
-    'm=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)\n' +
-    '})(window,document,\'script\',\'//www.google-analytics.com/analytics.js\',\'ga\');\n' +
-    'ga(\'create\', \'' + gaTrackingId + '\', \'auto\');\n' +
-    'ga(\'send\', \'pageview\');\n' +
-    '</script>\n';
-
-    $(head).append(analyticsHtml);
+  function injectAnalytics() {
+    var analyticsHtml = document.createElement("script");
```

Comment:
technical nit: this is not going to be "HTML", this is a script, so naming variables after what they are is still an excellent idea. Let's rename this to `analytics` or `gaScript` or something.